tests/test_generators/test_fhirgen.py

```diff
@@ -0,0 +1,42 @@
+"""Tests for: fhirgen: FHIR-related generators"""
+import unittest
+
+from linkml.generators.fhirgen import FhirValueSetGenerator
+from tests.test_generators.environment import env
+
+
+# TODO: Where to put output JSON?
```

Comment:
@hrshdhgd I haven't looked at where I should put my outputs yet, but I think I can figure that out easily.